client/components/mma/accountoverview/ProductCard.tsx

```diff
@@ -561,14 +561,29 @@ export const ProductCard = ({
 									<h4 css={sectionHeadingCss}>
 										Cancel {groupedProductType.friendlyName}
 									</h4>
-									<p
-										css={css`
-											max-width: 350px;
-										`}
-									>
-										Stop your recurring payment, at the end
-										of current billing period.
-									</p>
+									{!productDetail.subscription.autoRenew &&
+									!productDetail.subscription
+										.nextPaymentDate ? (
+										<p
+											css={css`
+												max-width: 350px;
+											`}
+										>
+											This is a one-off payment and will
+											not renew. You’ll continue to enjoy
+											your benefits until the end of the
+											current billing period.
+										</p>
+									) : (
+										<p
+											css={css`
+												max-width: 350px;
+											`}
+										>
+											Stop your recurring payment, at the
+											end of current billing period.
+										</p>
+									)}
```

Comment:
I think the styling is the same each way
```suggestion
									<p
										css={css`
											max-width: 350px;
										`}
									>
										{!productDetail.subscription.autoRenew &&
										!productDetail.subscription
											.nextPaymentDate ? (<>
												This is a one-off payment and will
												not renew. You’ll continue to enjoy
												your benefits until the end of the
												current billing period.
											</>) : (<>
												Stop your recurring payment, at the
												end of current billing period.
											</>)}
									</p>
```